lib/levels.js

```diff
@@ -33,7 +33,9 @@ const initialLsCache = Object.keys(nums).reduce((o, k) => {
 
 function genLsCache (instance) {
   instance[lsCacheSym] = Object.keys(instance.levels.labels).reduce((o, k) => {
-    if (!(k in o)) o[k] = flatstr('{"level":' + Number(k))
+    o[k] = instance[useLevelLabelsSym]
+      ? `{"level":"${instance.levels.labels[k]}"`
+      : flatstr('{"level":' + Number(k))
```

Comment:
by any chance, can we avoid adding another symbol and passing another argument to `getLsCache`?

Reply:
If I remove the symbol then I will have to pass the `useLevelLabels` option around to various parts of the code so that child loggers will have access to it.

Reply:
Ok!